src/max_cost.py

```diff
@@ -0,0 +1,9 @@
+'''Find max value in dict'''
+
+
+def max_cost(dict_):
+    return max(dict_.items(), key=lambda x: x[-1])
```

Comment:
do we really need this lambda?

Reply:
Yes, if  I don't use lambda program return a max key.

Reply:
Yes, if I don't use the lambda function, the function return a max key of dict.